pywbem/cim_operations.py

```diff
@@ -482,11 +534,26 @@ class WBEMConnection(object):
         :class:`~pywbem.BaseOperationRecorder`, each operation
         that is executed on this connection will be recorded by
         invoking its :meth:`~pywbem.BaseOperationRecorder.record` method.
```

Comment:
This is where we can add a statement that for the `Iter*()` methods, the recording will be done at the level of the lower level methods they invoke.